config/foreman.migrations/20210708144320_add_foreman_puppet.rb

```diff
@@ -0,0 +1,3 @@
+unless answers.key?('foreman::plugin::puppet')
+  answers['foreman::plugin::puppet'] = true
```

Comment:
do we still need this? this plugin is new in the installer, so we should not need the migrations now.

Reply:
The migrations provide a way for upgraded systems to get the new module into the answers file. Without a migration, upgraded users would not have this entry in their answers file. You can think of the answers file like a database, from version to version we release we use migrations to update that schema. The default answers we deliver is as if we delivered the end state schema of the database.

Reply:
Essentially: if the answers file is missing, packaging will create it (with the then latest default). After that it's all migrations.

Also to be explicit, the migrations do run on the placed answers file. That's why we now use `unless answers.key?('foreman::plugin::puppet')`. If we just used `||= true` it would break in the future if we do want to flip the default to `false`. The use of `||= false` would be safe.